h3-iroh/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "h3-iroh"
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+
+[dependencies]
+anyhow = { version = "1", optional = true }
+axum = { version = "0.7", optional = true }
+bytes = "1"
+futures = "0.3"
+h3 = { version = "0.0.6", features = ["tracing"] }
+http = { version =  "1.1", optional = true }
+http-body = { version = "1", optional = true }
+http-body-util = { version = "0.1", optional = true }
+hyper = { version = "1.5", optional = true }
+hyper-util = { version = "0.1", optional = true }
+iroh-net = { git = "http://github.com/n0-computer/iroh/", branch = "main" }
```

Comment:
@flub I think this won't work anymore 😅

Reply:
oops, yes: https://github.com/n0-computer/iroh-experiments/pull/21